evap/staff/templates/quick-review.js

```diff
@@ -164,6 +168,7 @@ $(document).ready(() => {
             type: "POST",
             url: "{% url 'staff:evaluation_textanswers_update_publish' %}",
             data: parameters,
+            success: function(data) { if(action == "textanswer_link" && data) window.location = data; },
```

Comment:
This seems a bit hacky, I don't know what we concluded the last time we talked about it. ~~We now thought that it could possibly be clearer if the button would be in a form by default (which would submit and redirect by default) and we only cancel the submission when we want to do ajax instead.~~

Even better: Always submit forms and use standard conventions to ignore the response.  According to https://stackoverflow.com/a/3283126/13679671 (I also tested with the evaluation_edit view) the browser will not reload the page if the status code is 204 - that means you could just return that for all cases except for the one where you really want to redirect. This means a small refactor is needed about the way these buttons work - I don't know how keen you are on HTML forms, but it probably shouldn't be too bad. 

If you don't want to spend more time on this, you can also leave it like this for now and we will fix it later :)